packages/client/components/GitHubScopingSearchInput.tsx

```diff
@@ -98,6 +99,7 @@ const GitHubScopingSearchInput = (props: Props) => {
   }
   const clearSearch = () => {
     setSearch(atmosphere, meetingId, '')
+    inputRef.current?.focus()
```

Comment:
After clearing a search input, I think we should keep the focus in the input as a user may want to continue searching